src/Rules/CanClaim.php

```diff
@@ -25,25 +26,22 @@ public function __construct(protected bool $singleUse = false)
      *
      * @param string $attribute
      * @param mixed  $value
+     * @param Closure $fail
      */
-    public function passes($attribute, $value): bool
+    public function validate(string $attribute, mixed $value, Closure $fail): void
     {
         if (!Arr::get($this->data, 'account')) {
-            return true;
+            return;
         }
 
         if ($this->singleUse) {
             $value = explode(':', decrypt($value), 3)[1];
         }
 
-        return ((int) Cache::get(BeamService::key($value), BeamService::claimsCountResolver($value))) > 0;
-    }
+        $passes = ((int) Cache::get(BeamService::key($value), BeamService::claimsCountResolver($value))) > 0;
 
-    /**
-     * Get the validation error message.
-     */
-    public function message(): string
-    {
-        return __('enjin-platform-beam::validation.can_claim');
+        if (!$passes) {
+            $fail(__('enjin-platform-beam::validation.can_claim'));
```

Comment:
I was wondering whether we should translate it this way as per documentation, either way works.
<img width="944" alt="Screenshot 2023-09-29 at 8 00 08 AM" src="https://github.com/enjin/platform-beam/assets/114082473/fafed135-dfeb-4426-9152-fbf4259f0eda">